src/shared/components/lollipopMutationPlot/Sequence.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import {SequenceSpec} from "./LollipopPlotNoTooltip";
+import {observer} from "mobx-react";
+import {observable} from "mobx";
+
+interface ISequenceProps {
+    x:number;
+    y:number;
+    width:number;
+    height:number;
+    color?:string;
+    hitzoneClassName?:string;
+    spec:SequenceSpec;
+}
+
+@observer
+export default class Sequence extends React.Component<ISequenceProps, {}> {
```

Comment:
This is the component for the gray area under the domains. `Sequence` was the legacy name for it, but it may not be the best name. Needed to add a separate component for that area to better handle tooltips...